docs/en/changes/changes.md

```diff
@@ -99,6 +99,7 @@
 * Refactor http-based alarm plugins and extract common logic to `HttpAlarmCallback`.
 * Support Amazon Simple Storage Service (Amazon S3) metrics monitoring
 * Support process Sum metrics with AGGREGATION_TEMPORALITY_DELTA case
+* Support prometheus HTTP API and promQL.
```

Comment:
```suggestion
* Support prometheus HTTP API and promQL.
* `Scope` in the Entity of Metrics query v1 protocol is not required and automatical correction. The scope is determined based on the metric itself.
```

I think we should update the GraphQL to remove required and add comments about the new mechanism.